ouroboros-network-api/src/Ouroboros/Network/AnchoredSeq.hs

```diff
@@ -295,6 +296,26 @@ splitAt i (AnchoredSeq a ft) = case FT.split (\v -> measureSize v > i) ft of
      let before' = AnchoredSeq a before
      in (before', AnchoredSeq (headAnchor before') after)
 
+-- | \( O(\log(\min(i,n-i)) \). Split the 'AnchoredSeq' at a given
+--  measure.
+--
+-- POSTCONDITION: @(before, after) = splitAtMeasure v s@, then:
+--
+-- > anchor      before == anchor s
+-- > headAnchor  before == anchor after
+-- > headAnchor  after  == headAnchor s
+-- > [before] == filter ((< v) . getElementMeasure . MeasuredWith) s
+-- > [after] == filter ((v <=) . getElementMeasure . MeasuredWith) s
```

Comment:
I think these two properties are false in boring ways when `s` is empty (as `filter` will then return an empty list for every predicate).

Reply:
Fixed.